src/scripts/themeLoader.js

```diff
@@ -16,6 +16,9 @@ define(['userSettings', 'skinManager', 'connectionManager', 'events'], function
                 context = 'serverdashboard';
             } else {
                 theme = userSettings.theme();
+                if (userSettings.followSystemTheme() === true && appHost.useDarkTheme() === true) {
```

Comment:
Is using explicitly `=== true` needed?

Reply:
`userSettings.followSystemTheme()` returns a bool so it should be fine with `=== true`

Reply:
Well yes I know, but you can omit `=== true`

Reply:
You can, but then it will accept any truthy value.

It's not strictly required, for sure, but it enforces a type (until we start moving to TypeScript, where this will be taken care of automatically).

Reply:
Okei that makes sense :+1: